src/java.base/share/classes/java/lang/invoke/MethodHandleProxies.java

```diff
@@ -202,7 +202,8 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
                     if (isObjectMethod(method))
                         return callObjectMethod(proxy, method, args);
                     if (isDefaultMethod(method)) {
-                        return callDefaultMethod(defaultMethodMap, proxy, intfc, method, args);
+                        // no additional access check is performed
+                        return JLRA.invokeDefault(proxy, method, args, null);
```

Comment:
As defined at the end of the file, `JavaLangReflectAccess JLRA = SharedSecrets.getJavaLangReflectAccess();`